src/pages/distroSettings/tabs/EventLogTab/useDistroEvents.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from "react";
+import { useQuery } from "@apollo/client";
+import { EVENT_LIMIT, useEvents } from "components/Settings/EventLog";
+import { useToastContext } from "context/toast";
+import {
+  DistroEventsQuery,
+  DistroEventsQueryVariables,
+} from "gql/generated/types";
+import { DISTRO_EVENTS } from "gql/queries";
+
+export const useDistroEvents = (
```

Comment:
I'm not sure if this is a me problem, but whenever I click "Load more" on the distro event log page I can't actually load any more events, even though I see them in the DB (tried it on local and on prod). The button works fine for me on the project event log page though 🤔